installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,227 @@
+
+[id="using-oc-mirror"]
+== Using oc-mirror
+
+The instructions in this section will show how to prepare the environment and use oc-mirror.
+
+.Environment Preparation
+
+* To set up the credentials for pulling/pushing container images, download the pull secret and place it in the correct path.
+
+** The default podman credentials location `$XDG_RUNTIME_DIR/containers/auth` is used for authenticating to the registries.
+** The docker location `~/.docker/config.json` for credentials is also supported.
+** Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
+
+[id="building-image-set-configuration"]
+== Building the image set configuration
+
+The Image Set Configuration is the input file used by oc-mirror to discover which images are necessary to be mirrored.
+
+Below is an example for the `ImageSetConfiguration` input file:
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+    - name: stable-4.13
+      minVersion: 4.13.10
+      maxVersion: 4.13.10
+    graph: true
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+      packages:
+       - name: aws-load-balancer-operator
+       - name: 3scale-operator
+       - name: node-observability-operator
+  additionalImages: 
+   - name: registry.redhat.io/ubi8/ubi:latest
+   - name: registry.redhat.io/ubi9/ubi@sha256:20f695d2a91352d4eaa25107535126727b5945bff38ed36a3e59590f495046f0
+----
+
+More Image Set Configuration examples can be found in the additional resources.
+
+[id="oc-mirror-workflows"]
+== oc-mirror workflows
+
+This section will explain all the workflows supported by oc-mirror currently.
+
+.Mirror to Disk
+
+Pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml <file path> --v2
+----
+
+.Disk to Mirror
+
+Copies the container images from the tar archive to a container registry.
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --from file:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Mirror To Mirror
+
+Copies the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --workspace file:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+[id="oc-mirror-delete"]
+== Image delete workflow in oc-mirror
```

Comment:
It is also important to mention in the Image delete workflow that oc-mirror only delete the manifests of the images (this does not reduce the storage being occupied in the registry). 

In order to reduce the storage of unnecessary images (the ones with manifests deleted) the customer needs to enable the garbage collector on their container registry (the procedure on doing that depends on which container registry they are using). Only with the garbage collector enabled the container registry will delete the images (blobs) which does not have reference to any manifest, reducing the storage previous occupied by the blobs deleted.

See the comment above as requested by you @kasturinarra.